### ISSUE TITLE
AppVeyor build currently failed due to timestamp format zero padding

### ISSUE THREAD

**Issue by dhermes** [GoogleCloudPlatform/google-cloud-python#2650]
See: https://ci.appveyor.com/project/GoogleCloudPlatform/google-cloud-python/build/1.0.850.master/job/v28e8ykrv4102h6f

I'm currently working on a fix.

**Comment by dhermes** [comment]
It's not just a Windows problem, so that's interesting.

```python
>>> from google.logging.v2 import log_entry_pb2
>>> from google.protobuf import json_format
>>> from google.protobuf import timestamp_pb2
>>> from google.cloud import _helpers
>>> 
>>> ts = timestamp_pb2.Timestamp(seconds=1477923316, nanos=283000000)
>>> entry = log_entry_pb2.LogEntry(timestamp=ts)
>>> e_dict = json_format.MessageToDict(entry)
>>> e_dict
{'timestamp': '2016-10-31T14:15:16.283Z'}
>>> _helpers._pb_timestamp_to_rfc3339(ts)
'2016-10-31T14:15:16.283000Z'
```

**Comment by dhermes** [comment]
It turns out this issue ["fixes itself"][1] in #2636 by actually comparing native types instead of strings

[1]: https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2636/files#diff-41b5ab9e98931dbeb0d6ea726a819a1eR298
